console/frontend/src/main/frontend/src/app/views/login/login.component.html

```diff
@@ -27,6 +27,7 @@ <h3>Frank<strong class="text-frank">!</strong>Console</h3>
           placeholder="Username"
           required
           autofocus
+          disabled
```

Comment:
?

Reply:
The form is currently not being used so I disabled the inputs. This way you only have to submit to go to the main view of the application